var/spack/repos/builtin/packages/py-jupyterlab/package.py

```diff
@@ -8,15 +8,36 @@ class PyJupyterlab(PythonPackage):
     """JupyterLab is the next-generation web-based user interface
        for Project Jupyter."""
 
-    homepage = "https://jupyterlab.readthedocs.io/"
+    homepage = "https://github.com/jupyterlab/jupyterlab"
     pypi = "jupyterlab/jupyterlab-2.2.7.tar.gz"
 
+    version('3.0.16', sha256='7ad4fbe1f6d38255869410fd151a8b15692a663ca97c0a8146b3f5c40e275c23')
     version('2.2.7', sha256='a72ffd0d919cba03a5ef8422bc92c3332a957ff97b0490494209c83ad93826da')
     version('2.1.0', sha256='8c239aababf5baa0b3d36e375fddeb9fd96f3a9a24a8cda098d6a414f5bbdc81')
 
+    depends_on('python@3.6:', when='@3:', type=('build', 'run'))
     depends_on('python@3.5:', type=('build', 'run'))
     depends_on('py-setuptools', type=('build', 'run'))
-    depends_on('py-notebook@4.3.1:', type=('build', 'run'))
-    depends_on('py-tornado@:5,6.0.3:', type=('build', 'run'))
-    depends_on('py-jupyterlab-server@1.1.5:1.999', type=('build', 'run'))
+    depends_on('py-jupyter-packaging@0.9:1.999', when='@3.0.15:', type=('build', 'run'))
```

Comment:
I don't see this dep in the setup.cfg

Reply:
Yes, I did not get it from the `setup.cfg` but the [`pyproject.toml`](https://github.com/jupyterlab/jupyterlab/blob/v3.0.16/pyproject.toml#L2)

Reply:
In that case I think this should be `type='build'`?

Reply:
Ok, I have changed it